packages/world/src/VoxelCoord.sol

```diff
@@ -317,12 +313,15 @@ library VoxelCoordLib {
       Position._set(entityId, coord.x, coord.y, coord.z);
       ReversePosition._set(coord.x, coord.y, coord.z, entityId);
       ObjectType._set(entityId, objectTypeId);
+      uint32 mass = ObjectTypeMetadata._getMass(objectTypeId);
+      if (mass > 0) {
```

Comment:
Won't this set the mass for relative entity ids when it shouldn't? Since this is being called in `BuildSystem`

Reply:
I think there's actually a test that checks that mass is only set on base entity id, so it should be failing

Reply:
hmm true, but I don't think we always get the base entity when checking the object type id, so it might be safe to do this?

Reply:
wait I don't think it does, as this is only for terrain entities no?

Reply:
yee you're right only terrain entities, so good for now then cuz we dont support non-voxel terrain blocks